config.toml

```diff
@@ -112,12 +112,26 @@ archived_version = false
 # The version number for the version of the docs represented in this doc set.
 # Used in the "version-banner" partial to display a version number for the
 # current doc set.
-version = "0.0"
+version = "v0.1.1"
 
 # A link to latest version of the docs. Used in the "version-banner" partial to
 # point people to the main doc site.
 url_latest_version = "https://cdevents.dev/docs"
 
+# Maintain history of release versions here
+# This section controls the content of the Releases menu
+[[params.versions]]
+  version = "latest"
+  url = "https://cdevents.dev/docs"
+
+[[params.versions]]
+  version = "v0.1.1"
+  url = "https://github.com/cdevents/spec/tree/v0.1.1"
+
+[[params.versions]]
+  version = "v0.1.0"
+  url = "https://github.com/cdevents/spec/tree/v0.1.0"
```

Comment:
We might want to only display the latest patch version of every `major.minor` series (we only have 0.1 today)

Reply:
Yes, the current setup was really just to illustrate how the menu works whilst we are transitioning.

Reply:
@afrittoli , should we then add an additional tag to the spec repo for each major.minor version, which we the move on to the latest patch version for every patch release?